### PR TITLE
Image input

### PR DESCRIPTION
**Pull request by aman11s**
- Added Image and Input component.
i. Responsive image
ii. Rounded image
iii. Textbox basic input
iv. Input with validations

- Netlify preview link is available below.

### CODE REVIEW

File: components/input.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
+.input-box {
+    padding: 0.5rem 1rem;
+    margin: 0.5rem 0;
```
Comment (VishalPatil18) [inline components/input.css:3]: Something like 0.5rem is quite small in my opinion when calculated in absolute value. So you could give them absolute values directly, as giving them such a low value in rems won't change them much on smaller screens also.

File: docsite.html [hunk 2]
```diff
@@ -488,18 +492,19 @@ <h2 class="h2">Floating Action Button</h2>
                 <p class="py-2 small-text">
                     A floating action button (FAB) performs the primary, or most common, action on a screen. It appears
                     in front of all screen content, typically as a circular shape with an icon in its center.
-                    You can use FAB by adding <span class="primary-text-color">"btn-fab"</span> as child class name inside anchor parent tag.
+                    You can use FAB by adding <span class="primary-text-color">"btn-fab"</span> as child class name
+                    inside anchor parent tag.
```
Comment (VishalPatil18) [inline docsite.html:496]: Code indentation could be improved here in this way, as that helps in code readability and maintainability.

```HTML
<p class="py-2 small-text">
    A floating action button (FAB) performs the primary, or most common, action on a screen.
    It appears in front of all screen content, typically as a circular shape with an icon in its center.
    You can use FAB by adding 
    <span class="primary-text-color">"btn-fab"</span>
    as child class name inside anchor parent tag.
</p>
```

File: docsite.html [hunk 3]
```diff
@@ -946,6 +954,155 @@ <h2 class="h2 my-4">Card with shadow</h2>
...
+
+                            &lt;label for="disabled">Disable&lt;/label>
+                            &lt;input type="text" id="disabled" disabled class="input-box input-disabled" placeholder="Disabled state">
+                        </code>
+                    </pre>
+                </div>
+            </div>
+
+            <hr class="my-4 section-hr">
+
```
Comment (VishalPatil18) [inline docsite.html:1105]: You could avoid multiple empty lines.

**Review by Husain01 — APPROVED** [review]
Looks good to go

**Review by VishalPatil18 — APPROVED** [review]
- As this `components/Images/.DS_Store`  file is not a part of the project you could avoid adding it here by including its path in the `.gitignore` file.
- A few suggestions were added regarding code indentation
- If you've not worked on responsiveness, then ignore this but if you have then your sidebar is getting overlapped with the content. Make sure to fix that.
- You could make the navbar fixed as that helps in better UX.

> Apart from these the rest looks quite good. Great work @aman11s 👍

**Review by starver20 — APPROVED** [review]
All good.

File: docsite.html (hunk 2 above)
Comment (aman11s) [reply docsite.html:496]: Thanks for the feedback @VishalPatil18, I will on these points.

**Pull request merged**